spec/server.test.js

```diff
@@ -2,18 +2,18 @@ const request = require('supertest');
 const server = require('../server/app');
 
 test('testing jest', () => {
-	expect(true).toBe(true);
+  expect(true).toBe(true);
 });
 
 describe('Test for path', () => {
-	test('It should response to GET method', () => {
-		return request(server).get('/rooms/4/reviews').then(response => {
-			expect(response.statusCode).toBe(201);
-		});
-	});
-	test('Data should exist', () => {
-		return request(server).get('/rooms/1/reviews').then(response => {
-			expect(response).toBeDefined();
-		});
-	});
+  test('It should response to GET method', () => {
+    return request(server).get('/rooms/4/reviews').then(response => {
+      expect(response.statusCode).toBe(201);
+    });
+  });
+  test('Data should exist', () => {
+    return request(server).get('/rooms/1/reviews').then(response => {
+      expect(response).toBeDefined();
+    });
+  });
 });
```

Comment:
maybe add a test where you are seeing if the data returned matches the structure/format that you are expecting?

Reply:
will be added! I'm working on tests on different branch right now! Thanks!

Reply:
cool! sounds good 👍